docs/partitioning.md

```diff
@@ -1,52 +1,150 @@
-# Partitioning output tables
-
+# Reducing cost of queries on BigQuery
 The [pricing model of BigQuery](https://cloud.google.com/bigquery/pricing#queries)
 is based on the total data processed in the columns selected by each query
 regardless of filtering criteria specified by `WHERE` or `LIMIT` conditions.
-As a result, partitioning the output table can reduce query costs significantly
-if only a particular region of the genome is being processed.
+For example, if you want to process only variants of chromosome 1 in a table
+containing entire genome variants, you would write a query like:
 
-Let us elaborate this using a more concrete example. If you want to query
-variants in chromosome 1 from a BigQuery table that contains variants from
-the entire genome, you would write a query like:
 ```
 SELECT ...
 FROM imported_table
 WHERE reference_name = 'chr1' AND ...
 ```
 
-In the above example, BigQuery actually processes all variants even though the
-query only considers variants on chromosome 1. You would also get charged for
-the entire size of the columns accessed in the SELECT clause. With partitioning,
-the data would be split into smaller tables, each containing variants of one
-chromosome. In that case the above query can be written as:
+In this example BigQuery processes all variants even though the query is
+limited to the variants on chromosome 1. This query will cost the entire size
+of the columns being accessed in the SELECT clause, regardless of the `WHERE`
+clause. This extra cost can add up to a significant amount, specially if a few
+*hot spots* in the genome are being queried regularly.
+
+We are offering three solutions for situations like this: one solution is using
+BigQuery [clustering](https://cloud.google.com/bigquery/docs/clustered-tables),
+another solution is using Variant Transforms' native partitioning, and a third
+hybrid solution.
+
+## Solution 1: BigQuery Clustering
```

Comment:
nit: s/Clustering/clustering (for consistency with other headers).

Reply:
done.